modules/login/src/main/resources/META-INF/resources/login/js/registration.js

```diff
@@ -0,0 +1,401 @@
+var registration = function($) {
+    var orgLookupData = {};
+    var bInitialized = false;
+
+    var $formStep1, $formStep2,$formStep3,$formStep4; //Form selector cache.
+    var $dlgStep1, $dlgStep2,$dlgStep3,$dlgStep4,$dlgConfirmation,$dlgError; //Dialog selector cache.
```

Comment:
Preference again here, but, I prefer to see words spelled out unless their is a compelling reason to abbreviate.